### ISSUE TITLE
Add password astericks when the auth token is saved

### ISSUE THREAD

**Issue by SteadyCadence** [Cadasta/cadasta-qgis-plugin#254]
The URL and username are saved when the auth token is saved. We should be consistent and have the password also filled in with astericks.  That removes confusion from users. 

I will also work on updating the wording so that it is more clear.

**Comment by dimasciput** [comment]
I don't think filled in password is a good idea for security purpose, we didn't store the password, we just store the username. We could add the asterick.

**Comment by dimasciput** [comment]
@SteadyCadence

**Comment by amplifi** [comment]
@dimasciput Agreed; we should not be storing or caching user passwords anywhere.

**Comment by SteadyCadence** [comment]
I agree.  Let's just put in 10 astericks or something.  

In other news, I like the greyed out fields when the username and password are stored.  Looks great!
